winch/codegen/src/isa/x64/masm.rs

```diff
@@ -1145,27 +1141,28 @@ impl Masm for MacroAssembler {
 }
 
 impl MacroAssembler {
+    /// Memory flags for trusted loads/stores.
+    const TRUSTED_FLAGS: MemFlags = MemFlags::trusted();
+
+    /// Flags used for WebAssembly loads / stores.
+    /// Untrusted by default so we don't set `no_trap`.
+    /// We also ensure that the endianess is the right one for WebAssembly.
+    const UNTRUSTED_FLAGS: MemFlags = MemFlags::new().with_endianness(Endianness::Little);
+
```

Comment:
When I was looking at this, I had thought of defining them where the `MacroAssesmbler` trait is defined, because they'll probably be consumed by all the other ISAs. Do you think it makes sense to move them over?

Reply:
That's a great idea! I'll move them over :+1: